pkg/blackduck/ctl_blackduck.go

```diff
@@ -267,29 +277,30 @@ func (ctl *HelmValuesFromCobraFlags) AddHelmValueByCobraFlag(f *pflag.Flag) {
 			util.SetHelmValueInMap(ctl.args, []string{"enableLivenessProbe"}, strings.ToUpper(ctl.flagTree.LivenessProbes) == "TRUE")
 		case "persistent-storage":
 			util.SetHelmValueInMap(ctl.args, []string{"enablePersistentStorage"}, strings.ToUpper(ctl.flagTree.PersistentStorage) == "TRUE")
-		//case "pvc-file-path":
-		//	data, err := util.ReadFileData(ctl.PVCFilePath)
-		//	if err != nil {
-		//		log.Fatalf("failed to read pvc file: %+v", err)
-		//	}
-		//	pvcs := []blackduckv1.PVC{}
-		//	err = json.Unmarshal([]byte(data), &pvcs)
-		//	if err != nil {
-		//		log.Fatalf("failed to unmarshal pvc structs: %+v", err)
-		//	}
-		//	for _, newPVC := range pvcs {
-		//		found := false
-		//		for i, currPVC := range ctl.blackDuckSpec.PVC {
-		//			if newPVC.Name == currPVC.Name {
-		//				ctl.blackDuckSpec.PVC[i] = newPVC
-		//				found = true
-		//				break
-		//			}
-		//		}
-		//		if !found {
-		//			ctl.blackDuckSpec.PVC = append(ctl.blackDuckSpec.PVC, newPVC)
-		//		}
-		//	}
+		case "pvc-file-path":
+			data, err := util.ReadFileData(ctl.flagTree.PVCFilePath)
+			if err != nil {
+				log.Fatalf("failed to read pvc file: %+v", err)
+			}
+			pvcs := []blackduckv1.PVC{}
+			err = json.Unmarshal([]byte(data), &pvcs)
+			if err != nil {
+				log.Fatalf("failed to unmarshal pvc structs: %+v", err)
+			}
+			pvcMappingOperatorToHelm := map[string]string{
```

Comment:
This would force us to release a new version of synopsysctl every time there is a new PVC. Can we default to pvc.name if not in pvcMappingOperatorToHelm?

Reply:
Nice idea!!!

Reply:
done @jdartigalongue